### DISCUSSION TITLE
[BUG] Opening a disabled repo on the app make it freeze

### DISCUSSION THREAD

**Discussion by EDM115** [community/community#63678] [Mobile]
### Select Topic Area

Bug

### Body

How to reproduce :  
- Have the GitHub app installed
- Click on a link that leads to a disabled repo
- If the app isn't opened, the app containing the source link will seems frozen (on the tasks, the GitHub app is here but empty)
- If the app was opened already, it will appear but frozen
  
I believe that it should also occur for taken down repos

**Comment by eliperkins** [comment]
Hey @EDM115, thanks for this report!

Can I get some info from you to help debug?

- What version of the app are you experiencing this on (e.g., GitHub for Android v1.125.0, GitHub for iOS v1.101.0)
- If you're on Android, can you confirm that tapping the Back button after this happens takes no action?

Thanks for helping us improve GitHub Mobile with bug reports like these!

**Reply by EDM115** [reply]
Android v1.126.0, and the back button actually unfroze the app

**Reply by eliperkins** [reply]
Great! Thanks for that info. I've filed an issue in our internal repo to track this issue. I'll follow up with our engineering team to dive in here, and get back to you as soon as we have a fix.
